nipype/interfaces/mrtrix3/reconst.py

```diff
@@ -187,6 +203,9 @@ def _list_outputs(self):
             outputs["gm_odf"] = op.abspath(self.inputs.gm_odf)
         if self.inputs.csf_odf != Undefined:
             outputs["csf_odf"] = op.abspath(self.inputs.csf_odf)
+        if self.inputs.predicted_signal != Undefined:
```

Comment:
I assume that you're being consistent with the rest of this function. If you feel the urge, you can fix up the others, but I think we should do this the standard way.

```suggestion
        if isdefined(self.inputs.predicted_signal):
```

Reply:
On this interface, I remembered that 'predicted_signal' is only a valid option when using the 'msmt_csd' algorithm. Is there a way to specify this in the input spec?